proposals/wg-freebsd-runtime.md

```diff
@@ -0,0 +1,78 @@
+# Working Group Proposal: FreeBSD Runtime
+
+Proposal created from [OCI WG template](https://github.com/opencontainers/tob/blob/master/WG-TEMPLATE.md).
+
+# FreeBSD Runtime OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the {WG NAME} Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+With two working OCI runtimes for FreeBSD
+([runj](https://github.com/samuelkarp/runj),
+[ocijail](https://github.com/dfr/ocijail)) and several container engines
+including containerd, podman and cri-o, there is a need to define a
+FreeBSD-specific section of the runtime-spec to allow support for platform
+features such as resource limits and fine-grained jail permissions. This will
+help to ensure runtime compatibility and build a consensus for the best way to
+support FreeBSD container runtimes.
+
+## Scope
+
+* Define a FreeBSD extension to the runtime-spec similar in scope to the
+  existing extensions for linux, solaris, windows etc.
+  * Identify platform features which should be part of the FreeBSD spec.
+  * Build consensus on how to represent these features clearly.
```

Comment:
nit: I'm not sure this line makes sense in the context of "scope", but we can keep it if you like.

Reply:
The intent here was there where there are multiple ways to represent a FreeBSD feature, we use the working group to resolve differences of opionion and settle on a single approach. Perhaps that doesn't need to be explcit in this document?